### PR TITLE
Update Functions Example App to CLI 3 Template

### PR DESCRIPTION
**Pull request by lizkenyon**

### CODE REVIEW

File: web/frontend/pages/volume/new.jsx [hunk 1]
```diff
@@ -24,14 +24,13 @@ import {
...
+import { data } from "@shopify/app-bridge/actions/Modal";
 import { useAuthenticatedFetch } from "../../hooks";
 
 const todaysDate = new Date();
-const METAFIELD_NAMESPACE = "YOUR_NAMESPACE";
+const METAFIELD_NAMESPACE = "FUNCTION_UPDATE_APP";
 const METAFIELD_CONFIGURATION_KEY = "volume-config";
-// REPLACE THE LINE BELOW WITH THE ID OF YOUR FUNCTION
-const FUNCTION_ID = "YOUR_VOLUME_FUNCTION_ID";
+const FUNCTION_ID = "01GQ0JMCSMAMSZTRHCH2RNGK0J";
```
Comment (scottdixon) [inline web/frontend/pages/volume/new.jsx]: Not sure if including your ID was intentional?

File: extensions/volume/input.graphql [hunk 2]
```diff
@@ -10,7 +10,7 @@ query Input {
     }
   }
   discountNode {
-    metafield(namespace: "YOUR_NAMESPACE", key: "volume-config") {
+    metafield(namespace: "FUNCTION_UPDATE_APP", key: "volume-config") {
```
Comment (scottdixon) [inline extensions/volume/input.graphql]: Nit: in the workshop this is `FUNCTION_APP` but either works 🤷

File: web/frontend/pages/volume/new.jsx [hunk 3]
```diff
@@ -24,14 +24,13 @@ import {
   Stack,
   PageActions,
 } from "@shopify/polaris";
-
+import { data } from "@shopify/app-bridge/actions/Modal";
 import { useAuthenticatedFetch } from "../../hooks";
 
 const todaysDate = new Date();
-const METAFIELD_NAMESPACE = "YOUR_NAMESPACE";
+const METAFIELD_NAMESPACE = "FUNCTION_UPDATE_APP";
```
Comment (scottdixon) [inline web/frontend/pages/volume/new.jsx]: Same here

**Review by scottdixon — APPROVED** [review]
🔥

**Review by Jan0707 — APPROVED** [review]
Nothing to complain about 🙌

But it _seems_ like you mixed two things here:
* the actual update for CLI3 and
* a general linting / cleaning up.

Ideally, we should be doing this in two separate PRs to make review easier.

**Comment by lizkenyon** [comment]
> Nothing to complain about 🙌
> 
> But it _seems_ like you mixed two things here:
> 
> * the actual update for CLI3 and
> * a general linting / cleaning up.
> 
> Ideally, we should be doing this in two separate PRs to make review easier.

@Jan0707 What is did is even worse. I generally I am very pro breaking changes up into many PRs, but I was feeling lazy, and kept it all in one. 🙈

**Comment by Jan0707** [comment]
@lizkenyon We have all been there 🤗 

Solid update either way!

**Pull request merged**